Sources/Testing/CMakeLists.txt

```diff
@@ -96,6 +96,14 @@ add_library(Testing
   Traits/Trait.swift)
 target_link_libraries(Testing PRIVATE
   _TestingInternals)
+if(NOT APPLE)
```

Comment:
Curious why we need the `if(NOT APPLE)` guard for these `target_link_libraries`? I get why it's needed for `find_package`, but don't we want to link Foundation even on Darwin?

Reply:
In Darwin, Foundation is a framework, so `-lFoundation` doesn't work.
Yes it links Foundation, but that's done via auto-linking. I.e. `import Foundation` automatically sets required linker flags.